### PR TITLE
Add Supported Platforms for some of the plugins

### PR DESCRIPTION
**Pull request by schmikei**
Bumping versions so the metadata gets updated on resources that track the next release.

### CODE REVIEW

File: plugins/mysql.yaml [hunk 1]
```diff
@@ -1,6 +1,12 @@
-version: 0.0.10
+version: 0.0.11
```
Comment (schmikei) [inline plugins/mysql.yaml:1]: Oops seems like we haven't released with 0.0.10, I'll just incorporate my change with that version.

**Review by ericwholt — COMMENTED** [review]
Please update changelog with this pr and add the changes to the unreleased section, but otherwise looks good.

File: plugins/mysql.yaml (hunk 1 above)
Comment (ericwholt) [reply plugins/mysql.yaml]: Good catch. I had bumped mysql version but haven't released. I had forgotten to bump with last release.

File: plugins/nginx.yaml [hunk 2]
```diff
@@ -9,6 +12,7 @@ parameters:
     valid_values:
       - file
       - kubernetes
+      - openshift
```
Comment (andykellr) [inline plugins/nginx.yaml:15]: Are we actually using this as a value?

File: plugins/nginx.yaml [hunk 3]
```diff
@@ -1,6 +1,9 @@
-version: 0.0.4
+version: 0.0.5
 title: Nginx
 description: Log parser for Nginx
+supported_platforms: 
+  - kubernetes
+  - openshift
```
Comment (andykellr) [inline plugins/nginx.yaml:6]: Isn't nginx still supported in linux, windows, and macos?

**Review by ericwholt — APPROVED** [review]

**Pull request merged**